Calculator/Calculator/Controller/CalculatorViewController.swift

```diff
@@ -55,61 +55,49 @@ class CalculatorViewController: UIViewController {
         guard let inputNumberText = sender.titleLabel?.text,
               let numberLabelText = inputNumberLabel.text else { return }
         
-        if numberLabelText.count < 20 {
-            if numberLabelText == "0" {
-                inputNumberLabel.text = "0"
+        let numberLabelTextWithoutComma = numberLabelText.replacingOccurrences(of: CalculatorNamespace.Comma,
+                                                                               with: CalculatorNamespace.Empty)
+        
+        if numberLabelText.count < 19 {
+            let formattedNumberText = Double(numberLabelTextWithoutComma + inputNumberText)
+            if numberLabelText == CalculatorNamespace.Zero {
+                inputNumberLabel.text = CalculatorNamespace.Zero
             } else {
-                let formattedNumberText = numberLabelText.replacingOccurrences(of: ",", with: "") + inputNumberText
-                inputNumberLabel.text = calculatorNumberFormatter.string(for: Double(formattedNumberText))
+                inputNumberLabel.text = calculatorNumberFormatter.string(for: formattedNumberText)
             }
         }
     }
     
     @IBAction func tapPeriodButton(_ sender: UIButton) {
-        let period = "."
+        let period = CalculatorNamespace.Period
         guard let numberLabelText = inputNumberLabel.text else { return }
         
-        inputNumberLabel.text? = (numberLabelText.contains(period)) ? (numberLabelText) : (numberLabelText + period)
+        inputNumberLabel.text =
+        (numberLabelText.contains(period)) ?
+        (numberLabelText) :
+        (numberLabelText + period)
```

Comment:
`.`이 있는 경우 다시 label.text에 기존의 숫자값을 할당하고 있어요. 조금 불필요한 과정이라는 생각이 들어요!
`.`이 있는 경우 리턴하게 되는게 더 좋다는 생각을 합니다.

```suggestion
        guard numberLabelText.contains(period) == false else { return }
        
        inputNumberLabel.text = numberLabelText + period
```

Reply:
코멘트 참고하여 수정하였습니다!😆